hail/build.gradle

```diff
@@ -31,7 +31,13 @@ buildScan {
     licenseAgree = 'yes'
 }
 
-String hailVersion = "0.2"
+// Hail has three notions of version:
+//  - hail short version: MAJOR.MINOR
+//  - hail pip version: MAJOR.MINOR.PATCH
+//  - hail version: MAJOR.MINOR.PATCH-GIT_SHA, calculated by generateBuildInfo
+String hailShortVersion = "0.2"
+// By default pip hides versions ending in ".devN". Use these for testing.
+String hailPipVersion = hailShortVersion + ".4" + System.getProperty("hail.pip-version-suffix", "")
```

Comment:
I don't like calling it "pip version". This is the full Major.Minor.Patch version of Hail, whether we deploy it to pip or conda or wherever.

Reply:
I wanted to call it `hailFullVersion` or `hailLongVersion` but both of those seem like they should refer to the version with the git sha. I'm at a loss for an appropriate name for this.

Reply:
patch version?